int16[] indices
float32[] values
